ambry-store/src/main/java/com.github.ambry.store/StoreManager.java

```diff
@@ -80,9 +81,11 @@ public void start()
       if (!file.exists()) {
         throw new IllegalStateException("Mount path does not exist " + replica.getMountPath());
       }
+      // Partition id is used as the id of a store.
+      String storeId = replica.getPartitionId().toString();
```

Comment:
`toString()` results in something like "Partition[0]" and not "0". Is that intentional?
Also that is the `toString()` as defined by an implementation.

Would it be possible to create the store id from `getBytes()` (which seems to be the unique id method in PartitionId interface)?